daffodil-lib/src/main/scala/org/apache/daffodil/cookers/Cookers.scala

```diff
@@ -63,7 +63,7 @@ object InitiatorCooker extends DelimiterCooker("initiator")
 
 object TerminatorCooker extends DelimiterCooker("terminator")
 
-object TerminatorCookerNoES extends DelimiterCookerNoES("terminator")
+object TerminatorCookerNoES extends DelimiterCookerNoSoleES("terminator")
```

Comment:
Is it appropriate for `TerminatorCookerNoES` to keep its original name or should it be renamed to `TerminatorCookieNoSoleES`?

Reply:
Good point. Maybe a more accurate name would be ``TerminatorDelimitedCooker``? Since that is what differentiates this cooker from the normal ``TerminatorCooker``--this one applies when lengthKind="delimited".